packages/developer-portal/src/components/apps/pipeline/pipeline-tasks-list.tsx

```diff
@@ -55,7 +64,7 @@ export const TaskList: FC<TaskListProps> = ({ tasks, s3BuildLogsLocation, buildS
       <div className={cx(elMt6)}>
         <Button
           intent="secondary"
-          disabled={!s3BuildLogsLocation || !['FAILED', 'SUCCEEDED'].includes(buildStatus)}
+          disabled={!s3BuildLogsLocation || !['FAILED', 'SUCCEEDED'].includes(buildStatus) || logsExpired}
```

Comment:
I think we should say, no logs available if they are expired or whatever? Maybe change the button text? Just begs the question why they aren't available if it's disabled